### ISSUE TITLE
Error of compiling Apollo6.0 in ubuntu18.04 with build_opt_gpu

### ISSUE THREAD

**Issue by zyq476916822** [ApolloAuto/apollo#14114]
Hi all, I tried to compile the newest version of Apollo in my ubuntu 18.04 and I would like to see how it performs in DreamView.

Here is the problem. As mentioned in the documentation, mainly 4 different type commands can be used for compiling:

$ bash ./apollo.sh build
$ bash ./apollo.sh build_opt
$ bash ./apollo.sh build_gpu
$ bash ./apollo.sh build_opt_gpu

(I did NOT compiled the modules of "perception" and "prediction")

I do have a NVIDIA graphics card and the required driver is properly installed and I truly believe build_opt_gpu would provides us the best performance in calculations (planning&prediction) and that's exactly what I was going to do. 

However, the DreamView cannot be boosted with the following errors:
$ ./scripts/bootstrap.sh start
nohup: appending output to 'nohup.out'
nohup: failed to run command 'cyber_launch': No such file or directory
Could not launch module monitor. Is it already built?
nohup: appending output to 'nohup.out'
nohup: failed to run command 'cyber_launch': No such file or directory
Could not launch module dreamview. Is it already built?

The funny thing is, when I re-compile it with "build_gpu" and "build_opt", then the DreanView can be successfully launched.

I literally have no idea what's wrong with it.
I will be so grateful if anyone could help with this.

**Comment by daohu527** [comment]
Could you pls provide printing logs during compilation. `build_opt_gpu` use for add `-c opt` to compile, seems it's not the problem.

**Comment by zyq476916822** [comment]
> Could you pls provide printing logs during compilation. `build_opt_gpu` use for add `-c opt` to compile, seems it's not the problem.

Hi, thank you for the reply. 
'build_opt_gpu [module]' is equivalent to 'bazel build --config=opt --config=gpu' 

I don't think I will make any changes.

**Comment by zyq476916822** [comment]
> Could you pls provide printing logs during compilation. `build_opt_gpu` use for add `-c opt` to compile, seems it's not the problem.

Hi, thank you for the reply. 
'build_opt_gpu [module]' is equivalent to 'bazel build --config=opt --config=gpu' 

I don't think I will make any changes.

**Comment by daohu527** [comment]
I did not ask you to change anything, I mean what is the cause of the problem, is there any printed information?

**Comment by zyq476916822** [comment]
I had done a bit more progress continuing on this issue. I ‘build_opt_gpu cyber’, and then this issue is figured out. 
However, the DreamView module still cannot be successfully launched.

I had looked into the log file and I compared it with successfully-launched log file. 
The log file was killed at “rate.cc:96] Detect forward jumps in time”
(Roughly around 500 lines)

So, the problem is no longer about compilation. DreamView just ends up with no reason. Or my computer couldn’t afford it. 
I really appreciate anyone who can help with this.
